src/AdvApi32.Desktop/AdvApi32.cs

```diff
@@ -218,7 +263,7 @@ public static partial class AdvApi32
         /// </returns>
         [DllImport(api_ms_win_service_management_l2_1_0, SetLastError = true, CharSet = CharSet.Unicode)]
         [return: MarshalAs(UnmanagedType.Bool)]
-        public static unsafe extern bool ChangeServiceConfig2(SafeServiceHandle hService, ServiceInfoLevel dwInfoLevel, void* lpInfo);
+        public static extern bool ChangeServiceConfig2(SafeServiceHandle hService, ServiceInfoLevel dwInfoLevel, IntPtr lpInfo);
```

Comment:
Well in the defense of `FriendlyFlags` when you don't know the size of the thing pointed by the target (So `void*`) there isn't much you can do 😺 The thing can't be stored in managed memory without more info so out, ref or even nullable are impossible.

But yes it's a bug in the generator it should mark such things as impossible